src/Draco.Editor.Web/app/index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
```

Comment:
I do not know conventions, are `head` and `body` not indented on purpose?

Reply:
I think the autoformat did that.

Reply:
Yes, it looks like vscode autoformatting use this convention.
Since these elements are always here, I'm not against it.